elytron/src/main/java/org/wildfly/extension/elytron/CredentialStoreResourceDefinition.java

```diff
@@ -192,7 +216,17 @@ private CredentialStoreAddHandler() {
         protected void performRuntime(OperationContext context, ModelNode operation, Resource resource) throws OperationFailedException {
 
             ModelNode model = resource.getModel();
-            String uri = asStringIfDefined(context, URI, model);
+            String location = asStringIfDefined(context, LOCATION, model);
+            boolean modifiable = asBooleanIfDefined(context, MODIFIABLE, model);
```

Comment:
asBooleanIfDefined calls attributeDefinition.resolveModelAttribute, which means if there is a default value it won't return undefined.